routes/cp.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use Illuminate\Support\Facades\Route;
+use Rapidez\StatamicQueryBuilder\Http\Controllers\CP\ProductAttributeController;
+
+Route::prefix('rapidez')->group(function () {
```

Comment:
Is there any auth on this route? Maybe add the logged in middleware from statamic?